config/locales/en.yml

```diff
@@ -1665,6 +1665,7 @@ en:
       multiple_errors: "Multiple fields violated their constraints."
       parse_error: "The request body was neither empty, nor did it contain a single JSON object."
       writing_read_only_attributes: "You must not write a read-only attribute."
+      invalid_format: "Invalid format for property '%{property}': Expected format like '%{expected_format}', but got '%{actual}'."
```

Comment:
Even worse: ISO 8601 allows more than this specific format... However, I thought it was better than writing:

> Expected format like 'ISO 8601'

I *wanted* to make the message generic enough to be reusable for other formating errors...

So better something like:

> Invalid format for property 'startDate': Expected input format of 'Date', but could not parse '2014-02-30' as such.

Would that be better?